Upgrade `gleam_stdlib` to `0.60.0`

awseward opened the pull request:
👋🏻 Hello!

First off, thanks for writing this library, I've found it helpful. I noticed a couple small issues I thought I'd fix an offer a PR for.

#### Here's a summary:

> Fix some deprecations:
> 
> 1. From [0.60.0](https://github.com/gleam-lang/stdlib/blob/main/CHANGELOG.md#v0600---2025-05-13):
> 
>     - The `from` function in the dynamic module has been deprecated.
> 
> 2. From [0.52.0](https://github.com/gleam-lang/stdlib/blob/main/CHANGELOG.md#v0520---2025-01-04):
> 
>     - The deprecated …, `result.nil_error`, … functions have been removed.

### Testing

I've got tests passing locally for all the target runtime on my macos environment, but I'm not sure how to validate other platforms.

awseward committed:
Upgrade `gleam_stdlib` to `0.60.0`

Fix some deprecations:

1. From https://github.com/gleam-lang/stdlib/blob/main/CHANGELOG.md#v0600---2025-05-13:

  - The from function in the dynamic module has been deprecated.

2. From https://github.com/gleam-lang/stdlib/blob/main/CHANGELOG.md#v0520---2025-01-04

  - The deprecated …, result.nil_error, … functions have been removed.

Olian04 commented:
Hey!
Thank you for the PR!

> I've got tests passing locally for all the target runtime on my macos environment, but I'm not sure how to validate other platforms.

If you could update table in the readme with an entry for the platform/platforms you've tested on

awseward committed:
Increment readme, udpate README testing table

awseward committed:
Replace `io.debug` with `echo`

>   Compiling stdin
> warning: Deprecated value used
>   ┌─ /Users/andrew/projects/gleam/gleam_stdin/test/stdin_test.gleam:8:9
>   │
> 8 │   |> io.debug
>   │         ^^^^^ This value has been deprecated
>
> It was deprecated with this message: To debug print a value use the `echo`
> keyword instead

awseward committed:
Update the README example, also

Referring to this error:

>   Compiling stdin
> warning: Deprecated value used
>   ┌─ /Users/andrew/projects/gleam/gleam_stdin/test/stdin_test.gleam:8:9
>   │
> 8 │   |> io.debug
>   │         ^^^^^ This value has been deprecated
>
> It was deprecated with this message: To debug print a value use the `echo`
> keyword instead

awseward commented:
Hi @Olian04, I've done that now; let me know if there's anything else I can do to get this mergeable.

Thanks!

awseward committed:
Update gleam_stdlib lower bound

I realized, the code changes I made require APIs only added into
gleam_stdlib as of v0.5.0, so this will help avoid errors if older
projects try updating.

Olian04 commented:
Thank you @awseward! I will look into releasing a new version tomorrow!

Olian04 commented:
Version 2.0.2 has been release. Thank you once again for your PR.
